tests/ert3/config/test_ert3_config.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import pydantic
+from ert3.config import _ensemble_config
+
+_config_dict = {
```

Comment:
Since you are not using this programatically, could you persist it as a YAML string instead. Easier to read perhaps 🤷

Reply:
It is used programmatically here: ` test_config():`, that test should perhaps be expanded a bit though.

Reply:
Yeah, I was more thinking that we could persist it as text and then use `yaml.loads` to make it into a dictionary. But it is fine to leave it as is 👍